pubmed_parser/pubmed_oa_parser.py

```diff
@@ -161,11 +161,11 @@ def parse_pubmed_xml(path, include_path=False, nxml=False):
         journal = ""
 
     dict_article_meta = parse_article_meta(tree)
-    pub_year_node = tree.find(".//pub-date[@pub-type='epub']/year")
+    pub_year_node = tree.find(".//pub-date/year")
     pub_year = pub_year_node.text if pub_year_node is not None else ""
-    pub_month_node = tree.find(".//pub-date[@pub-type='epub']/month")
+    pub_month_node = tree.find(".//pub-date/month")
     pub_month = pub_month_node.text if pub_month_node is not None else "01"
-    pub_day_node = tree.find(".//pub-date[@pub-type='epub']/day")
+    pub_day_node = tree.find(".//pub-date/day")
```

Comment:
This doesn't belong here.

Reply:
See [comment below](https://github.com/titipata/pubmed_parser/pull/142#discussion_r1611396874)